app/controllers/comments_controller.rb

```diff
@@ -1,28 +1,54 @@
 class CommentsController < ApplicationController
   before_action :authenticate_current_user!
   before_action :set_comment
+  before_action -> { authorize @comment }, only: %i[edit update destroy]
```

Comment:
```suggestion
  before_action -> { authorize @comment }, only: %i[edit update destroy new]
```
Why you do it in new action?

Reply:
create comment?

Reply:
new

Reply:
it is not principal, it strange)